pkgs/applications/misc/chataigne/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchurl, appimageTools, pkgs }:
+
+let
+  pname = "chataigne";
+  version = "1.8.0";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://benjamin.kuperberg.fr/chataigne/user/data/Chataigne-linux-x64-1.8.0.AppImage";
+    name="${pname}-${version}.AppImage";
+    sha512 = "aHo+8U0jRczJHQwMMmvNYpWdamensO5hpIIHgaayvvI+NFoN4Hb9VhVY9vJgv5OkzktjhfTs4WsB+SHUDjQF2Q==";
+  };
+  curlWithGnutls = pkgs.curl.override { gnutlsSupport = true; sslSupport = false; };
+in
+appimageTools.wrapType2 {
+  inherit name src;
```

Comment:
It's preferable to inherit `pname` and `version` instead of `name`. Doing so produces the same `name`, but the extra attributes help our tooling.